timex/timex_test.go

```diff
@@ -0,0 +1,45 @@
+package timex
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/sync/errgroup"
+)
+
+func TestTicker(t *testing.T) {
+	ticker := NewTicker(2 * time.Millisecond)
+	time.AfterFunc(5*time.Millisecond, func() {
```

Comment:
`func() { ticker.Stop() }` can be replaced with `ticker.Stop`

Reply:
done, nice! i never see these.

Reply:
`func() { ... }` is a nullary function. `ticker.Stop` is a nullary function. `time.AfterFunc` takes a nullary function. QED